### PR TITLE
Helm-388: Rename Helm in the Helm docs

### PR DESCRIPTION
**Pull request by Bonrob2**
# Pull Request Check Sheet

* [ ] Have you read and followed our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [ ] Have you [made an issue in the OpenNMS issue tracker](https://issues.opennms.org)?<br>If so, you should:
  1. update the title of this PR to be of the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the JIRA link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
* [ ] Have you made a comment in that issue which points back to this PR?
* [ ] Have you updated the JIRA link at the bottom of this comment to link to your issue?
* [ ] If this is a new feature, is there documentation?
* [ ] If this is a new feature or substantial change, are there tests that cover it?

# Pull Request Process

One or more reviewers should be assigned to each PR.

If you know that a particular person is subject matter expert in the area your PR affects, feel free to assign one or more reviewers when you create this PR, otherwise reviewers will be assigned for you.

Once the reviewer(s) accept the PR and the branch passes continuous integration in Bamboo, the PR is eligible for merge.

At that time, if you have commit access (are an OpenNMS Group employee or a member of the Order of the Green Polo) you are welcome to merge the PR.
Otherwise, a reviewer can merge it for you.

Thanks for taking time to contribute!

# External References

* JIRA (Issue Tracker): http://issues.opennms.org/browse/${JIRA-ISSUE-NUMBER}
* Continuous Integration: [CircleCI](https://circleci.com/gh/OpenNMS/opennms-helm)

### CODE REVIEW

**Review by emarsh1995 — APPROVED** [review]
LGTM! Thank you! :)

File: Dockerfile [hunk 1]
```diff
@@ -2,7 +2,7 @@ ARG GRAFANA_VERSION="latest"
 
 FROM grafana/grafana:${GRAFANA_VERSION}
 
-ARG OPENNMS_HELM_VERSION="bleeding"
+ARG OPENNMS_GRAFANA-PLUGIN_VERSION="bleeding"
```
Comment (synqotik) [inline Dockerfile:5]: Probably should be `OPENNMS_GRAFANA_PLUGIN_VERSION` (underscore instead of hyphen)

File: .circleci/config.yml [hunk 2]
```diff
@@ -344,9 +344,9 @@ jobs:
       - run:
           name: Tag Docker Container Images for release and publish to DockerHub
           command: |
-            HELM_IMAGE_NAME="$(cat ./helm-docker-image-name.txt)"
-            docker tag "${HELM_IMAGE_NAME}" ${DOCKERHUB_PROJECT_USER}/${DOCKERHUB_PROJECT_NAME}:$(.circleci/get-version.sh)
-            docker tag "${HELM_IMAGE_NAME}" ${DOCKERHUB_PROJECT_USER}/${DOCKERHUB_PROJECT_NAME}:latest
+            GRAFANA-PLUGIN_IMAGE_NAME="$(cat ./grafana-plugin-docker-image-name.txt)"
```
Comment (synqotik) [inline .circleci/config.yml:347]: Should be `GRAFANA_PLUGIN_IMAGE_NAME`

File: src/dashboards/flow_deep_dive.json [hunk 3]
```diff
@@ -1848,7 +1848,7 @@
         "multi": false,
         "name": "flowds",
         "options": [],
-        "query": "opennms-helm-flow-datasource",
+        "query": "grafana-plugin-flow-datasource",
```
Comment (synqotik) [inline src/dashboards/flow_deep_dive.json:1851]: Change to just `opennms-flow-datasource`, this is the name/id we are using.

File: src/dashboards/flow_deep_dive.json [hunk 4]
```diff
@@ -1864,7 +1864,7 @@
         "multi": false,
         "name": "perfds",
         "options": [],
-        "query": "opennms-helm-performance-datasource",
+        "query": "grafana-plugin-performance-datasource",
```
Comment (synqotik) [inline src/dashboards/flow_deep_dive.json:1867]: Change to `opennms-performance-datasource`

File: src/lib/dashboard-convert/alarmHistogramPanel.ts [hunk 5]
```diff
@@ -1,7 +1,7 @@
 import { AlarmDirections, AlarmGroups } from '../../panels/alarm-histogram/constants'
 
 export const isLegacyAlarmHistogramPanel = (panel: any) => {
-  return panel && panel.type && panel.type === 'opennms-helm-alarm-histogram-panel'
+  return panel && panel.type && panel.type === 'grafana-plugin-alarm-histogram-panel'
```
Comment (synqotik) [inline src/lib/dashboard-convert/alarmHistogramPanel.ts:4]: Change to `opennms-alarm-histogram-panel`

File: src/lib/dashboard-convert/alarmTablePanel.ts [hunk 6]
```diff
@@ -14,7 +14,7 @@ const legacyAlarmSeverityThemes = [
 ]
 
 export const isLegacyAlarmTablePanel = (panel: any) => {
-  return panel && panel.type && panel.type === 'opennms-helm-alarm-table-panel'
+  return panel && panel.type && panel.type === 'grafana-plugin-alarm-table-panel'
```
Comment (synqotik) [inline src/lib/dashboard-convert/alarmTablePanel.ts:17]: Change to `opennms-alarm-table-panel`

File: src/lib/dashboard-convert/entityDs.ts [hunk 7]
```diff
@@ -5,7 +5,7 @@ import { OnmsEntityNestType, OnmsEntityType } from '../../datasources/entity-ds/
 // Legacy Entity target looks something like this:
 // {
 //   "datasource": {
-//     "type": "opennms-helm-entity-datasource",
+//     "type": "grafana-plugin-entity-datasource",
```
Comment (synqotik) [inline src/lib/dashboard-convert/entityDs.ts:8]: Change to `opennms-entity-datasource`

File: src/lib/dashboard-convert/filterPanel.ts [hunk 8]
```diff
@@ -1,7 +1,7 @@
 import { DatasourceMetadata, DsType } from './types'
 
 export const isLegacyFilterPanel = (panel: any) => {
-  return panel && panel.type && panel.type === 'opennms-helm-filter-panel'
+  return panel && panel.type && panel.type === 'grafana-plugin-filter-panel'
```
Comment (synqotik) [inline src/lib/dashboard-convert/filterPanel.ts:4]: Change to `opennms-filter-panel`

File: src/lib/dashboard-convert/flowHistogramPanel.ts [hunk 9]
```diff
@@ -8,7 +8,7 @@ import {
 import { FlowHistogramOptionsProps } from '../../panels/flow-histogram/FlowHistogramTypes'
 
 export const isLegacyFlowHistogramPanel = (panel: any) => {
-  return panel && panel.type && panel.type === 'opennms-helm-flow-histogram-panel'
+  return panel && panel.type && panel.type === 'grafana-plugin-flow-histogram-panel'
```
Comment (synqotik) [inline src/lib/dashboard-convert/flowHistogramPanel.ts:11]: Change to `opennms-flow-histogram-panel`

File: src/lib/dashboard-convert/inputs.ts [hunk 10]
```diff
@@ -14,7 +14,7 @@ import { addVariationsToMap, getDatasourceTypeFromPluginId } from './utils'
 //     "label": "OpenNMS Performance",
 //     "description": "",
 //     "type": "datasource",
-//     "pluginId": "opennms-helm-performance-datasource",
+//     "pluginId": "grafana-plugin-performance-datasource",
```
Comment (synqotik) [inline src/lib/dashboard-convert/inputs.ts:17]: Should be `opennms-performance-datasource`

File: src/lib/dashboard-convert/templating.ts [hunk 11]
```diff
@@ -23,7 +23,7 @@ import { addVariationsToMap, getDatasourceTypeFromPluginId } from './utils'
 //       "multi": false,
 //       "name": "datasource",
 //       "options": [],
-//       "query": "opennms-helm-performance-datasource",
+//       "query": "grafana-plugin-performance-datasource",
```
Comment (synqotik) [inline src/lib/dashboard-convert/templating.ts:26]: Should be `opennms-performance-datasource`

File: src/lib/dashboard-convert/utils.ts [hunk 12]
```diff
@@ -12,7 +12,7 @@ export const addVariationsToMap = (varName: string, dsType: DsType,  datasourceM
 // returns a string which is one of the DsTypes or else empty string, and
 // whether this is a "legacy" datasource or not (legacy is version < 9)
 export const getDatasourceTypeFromPluginId = (pluginId: string) => {
-  const legacyMatch = pluginId.match(/^opennms-helm-([^-]+)-datasource$/i)
+  const legacyMatch = pluginId.match(/^grafana-plugin-([^-]+)-datasource$/i)
```
Comment (synqotik) [inline src/lib/dashboard-convert/utils.ts:15]: Should be `opennms-` instead of `opennms-helm-`

File: src/plugin.json [hunk 13]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "OpenNMS Plugin for Grafana",
-  "id": "opennms-helm-app",
+  "id": "grafana-plugin-app",
```
Comment (synqotik) [inline src/plugin.json:3]: I feel we should probably use `opennms-grafana-plugin-app`. We should have `opennms` in the ID, so as to identify the plugin as ours

File: docker-compose.yaml [hunk 14]
```diff
@@ -2,13 +2,13 @@ version: '3.0'
 
 services:
   grafana:
-    container_name: 'opennms-helm-app'
+    container_name: 'grafana-plugin-app'
```
Comment (synqotik) [inline docker-compose.yaml:5]: This should probably be `opennms-grafana-plugin-app`

File: docker-compose.yaml [hunk 15]
```diff
@@ -2,13 +2,13 @@ version: '3.0'
...
     build:
       context: ./.config
       args:
         grafana_version: ${GRAFANA_VERSION:-9.4.7}
     ports:
       - 3000:3000/tcp
     volumes:
-      - ./dist:/var/lib/grafana/plugins/opennms-helm-app
+      - ./dist:/var/lib/grafana/plugins/grafana-plugin-app
```
Comment (synqotik) [inline docker-compose.yaml:13]: This should probably be `opennms-grafana-plugin-app`

File: docker-compose.yml [hunk 16]
```diff
@@ -4,7 +4,7 @@ volumes:
     driver: "local"
   data-opennms:
     driver: "local"
-  data-opennms-helm:
+  data-grafana-plugin:
     driver: "local"
```
Comment (synqotik) [inline docker-compose.yml:8]: maybe `data-opennms-grafana-plugin`

File: docker-compose.yml [hunk 17]
```diff
@@ -56,18 +56,18 @@ services:
...
-  opennms-helm:
+  opennms-grafana-plugin:
     build:
       context: ./
       dockerfile: Dockerfile
       args:
-        OPENNMS_HELM_GIT_URL: https://github.com/OpenNMS/opennms-helm.git
-        OPENNMS_HELM_GIT_BRANCH_REF: master
-    image: opennms-helm:master
+        OPENNMS_GRAFANA-PLUGIN_GIT_URL: https://github.com/OpenNMS/grafana-plugin.git
```
Comment (synqotik) [inline docker-compose.yml:64]: -> `OPENNMS_GRAFANA_PLUGIN_GIT_URL`

File: docker-compose.yml [hunk 18]
```diff
@@ -56,18 +56,18 @@ services:
...
+        OPENNMS_GRAFANA-PLUGIN_BRANCH_REF: master
+    image: grafana-plugin:master
     volumes:
-      - data-opennms-helm:/var/lib/grafana
-      - data-opennms-helm:/var/log/grafana
-      - data-opennms-helm:/etc/grafana
+      - data-grafana-plugin:/var/lib/grafana
```
Comment (synqotik) [inline docker-compose.yml:68]: `data-opennms-grafana-plugin`

File: docker-compose.yml [hunk 19]
```diff
@@ -56,18 +56,18 @@ services:
...
+      - data-grafana-plugin:/var/log/grafana
```
Comment (synqotik) [inline docker-compose.yml:69]: `data-opennms-grafana-plugin`

File: makezip.js [hunk 20]
```diff
@@ -21,7 +21,7 @@ const topdir = process.cwd();
 
 const version = pkginfo.version;
 
-const pkgname = 'opennms-helm-app';
+const pkgname = 'grafana-plugin-app';
```
Comment (synqotik) [inline makezip.js:24]: maybe `opennms-grafana-plugin-app` here

File: makerpm.js [hunk 21]
```diff
@@ -73,7 +73,7 @@ generate(cwd, pkginfo, options, pkginfo.name, function (err, generated) {
     [
       '--define', '_topdir ' + path.join(os.tmpdir(), 'rpmbuild'),
       '-ba',
-      'SPECS/opennms-helm.spec'
+      'SPECS/grafana-plugin.spec'
```
Comment (synqotik) [inline makerpm.js:76]: `opennms-grafana-plugin.spec`

File: package.json [hunk 22]
```diff
@@ -1,17 +1,17 @@
 {
-  "name": "opennms-helm",
+  "name": "grafana-plugin",
```
Comment (synqotik) [inline package.json:2]: `opennms-grafana-plugin`

**Pull request merged**